<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDialog</class>
 <widget class="QDialog" name="GraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(255, 170, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_TargetAbout_target">
     <property name="title">
      <string>目标相关</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_insertnewGraph_target">
        <property name="text">
         <string>增加新的曲线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delchooseGraph_target">
        <property name="text">
         <string>删除选中曲线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_leftmovegraph_target">
        <property name="text">
         <string>左移曲线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_rightmovegraph_target">
        <property name="text">
         <string>右移曲线</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1_target">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>567</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox_OKorCancel_target">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2_target">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>881</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(255, 85, 0);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>曲线0</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_AxisConfig_tab">
        <property name="title">
         <string>坐标项配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_AxisConfig">
         <item>
          <widget class="QGroupBox" name="groupBox_xAxisConfig_tab">
           <property name="title">
            <string>X轴配置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_xAxisConfig">
            <item row="0" column="0">
             <widget class="QLabel" name="label_kedufanwei_x_tab">
              <property name="text">
               <string>刻度范围：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_Xmax_tab">
              <property name="text">
               <string>最大值</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Xmax_tab"/>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_1_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>48</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_Xmin_tab">
              <property name="text">
               <string>最小值</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Xmin_tab"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_wenbenpianyi_x_tab">
              <property name="text">
               <string>文本偏移：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer__3_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_wenbenpianyi_x"/>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_AxisLabel_x">
              <property name="text">
               <string>轴标签（名称及单位）：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_inputAxisLabel_x"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_keduweizhi_x">
              <property name="text">
               <string>刻度位置：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <spacer name="horizontalSpacer_5_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>48</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="comboBox_keduweizhi_x">
              <item>
               <property name="text">
                <string>上</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>下</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_Axisdisplay_x">
              <property name="text">
               <string>显示坐标轴：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QCheckBox" name="checkBox_chooseAxisdisplay_x">
              <property name="text">
               <string>默认选中显示X轴</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <spacer name="horizontalSpacer_7_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>31</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_yAxisConfig_tab">
           <property name="title">
            <string>Y轴配置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_yAxisconfig">
            <item row="0" column="0">
             <widget class="QLabel" name="label_kedufanwei_y">
              <property name="text">
               <string>刻度范围：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_Ymax">
              <property name="text">
               <string>最大值</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Ymax"/>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_Ymin">
              <property name="text">
               <string>最小值</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Ymin"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13_wenbenpianyi_y">
              <property name="text">
               <string>文本偏移：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer_4_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_wenbenpianyi_y"/>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_AxisLabel_y">
              <property name="text">
               <string>轴标签（名称及单位）：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit_inputAxisLabel_y"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_keduweizhi_y">
              <property name="text">
               <string>刻度位置：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <spacer name="horizontalSpacer_6_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2" colspan="2">
             <widget class="QComboBox" name="comboBox_keduweizhi_y">
              <item>
               <property name="text">
                <string>左</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>右</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_Axisdisplay_y">
              <property name="text">
               <string>显示坐标轴：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QCheckBox" name="checkBox_chooseAxisdisplay_y">
              <property name="text">
               <string>默认选中显示Y轴</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <spacer name="horizontalSpacer_8_tab">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>41</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
           <zorder>label_13_wenbenpianyi_y</zorder>
           <zorder>horizontalSpacer_4_tab</zorder>
           <zorder>label_Ymax</zorder>
           <zorder>label_Axisdisplay_y</zorder>
           <zorder>horizontalSpacer_2_tab</zorder>
           <zorder>label_AxisLabel_y</zorder>
           <zorder>label_Ymin</zorder>
           <zorder>label_kedufanwei_y</zorder>
           <zorder>horizontalSpacer_6_tab</zorder>
           <zorder>label_keduweizhi_y</zorder>
           <zorder>lineEdit_Ymin</zorder>
           <zorder>lineEdit_wenbenpianyi_y</zorder>
           <zorder>lineEdit_inputAxisLabel_y</zorder>
           <zorder>comboBox_keduweizhi_y</zorder>
           <zorder>checkBox_chooseAxisdisplay_y</zorder>
           <zorder>horizontalSpacer_8_tab</zorder>
           <zorder>lineEdit_Ymax</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9_tab">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_GraphConfig_tab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>181</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>181</height>
         </size>
        </property>
        <property name="title">
         <string>曲线自身配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_Graphconfig">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_BaseCharacter_tab">
             <property name="title">
              <string>基本特征</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_GraphName_tab">
                <property name="text">
                 <string>曲线名称（图例名称）：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_inputGraphName_tab"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_GraphColor_tab">
                <property name="text">
                 <string>曲线颜色：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_GraphWidth_tab">
                <property name="text">
                 <string>曲线宽度：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_GraphWidth_tab"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_GraphBuffer_tab">
                <property name="text">
                 <string>曲线缓冲区：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_GraphBuffer_tab"/>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_chooseGraphColor">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="pushButton_chooseGraphColor_tab">
                <property name="text">
                 <string>选择曲线颜色</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_Data_tab">
             <property name="title">
              <string>数据源相关</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_Data">
              <item row="1" column="4">
               <widget class="QPushButton" name="pushButton_chooseXparam_tab">
                <property name="text">
                 <string>选择X参数</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="3">
               <widget class="QLineEdit" name="lineEdit_inputXparam_tab"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_Xparam_tab">
                <property name="text">
                 <string>X参数</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4" colspan="2">
               <widget class="QPushButton" name="pushButton_chooseT0_tab">
                <property name="text">
                 <string>选择T0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_inputT0_tab">
                <property name="text">
                 <string>输入T0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_chooseGraphbasemap_tab">
                <property name="text">
                 <string>选择弹道底图（理论弹道）</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QLineEdit" name="lineEdit_inputYparam_tab"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_Yparam_tab">
                <property name="text">
                 <string>Y参数</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="pushButton_chooseYparam_tab">
                <property name="text">
                 <string>选择Y参数</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
